server/controllers/dashboard/debtorGroups.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const moment = require('moment');
+
+const report = require('../finance/reports/agedDebtors');
+
+// time until the cached value expires - currently set at 5 minutes
+const CACHE_TTL = 300000;
```

Comment:
It could also be stored using redis or mysql but I think this is okay for now - the main consideration is that this does not scale well across different server machines or cores.